.github/workflows/zip-templates-and-add-to-release.yml

```diff
@@ -0,0 +1,33 @@
+name: zip-templates-and-add-to-release
+on:
+  release:
+    types: [published]
+jobs:
+  add-templates-to-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```

Comment:
This may not be needed as Node version 14 is installed in the ubuntu image being used - https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#language-and-runtime

Reply:
I tried removing this and got a permissions error when running it without this